src/main/java/com/google/api/codegen/config/DiscoveryMethodModel.java

```diff
@@ -187,12 +190,16 @@ public String getAndSaveRequestTypeName(ImportTypeTable typeTable, SurfaceNamer
 
   @Override
   public String getAndSaveResponseTypeName(ImportTypeTable typeTable, SurfaceNamer surfaceNamer) {
-    TypeName fullName =
-        typeTable
-            .getTypeTable()
-            .getTypeNameInImplicitPackage(
-                surfaceNamer.publicClassName(DiscoGapicNamer.getResponseName(method)));
-    return typeTable.getAndSaveNicknameFor(fullName.getFullName());
+    Name responseName = DiscoGapicNamer.getResponseName(method);
+    if (responseName != null) {
+      TypeName fullName =
+          typeTable
+              .getTypeTable()
+              .getTypeNameInImplicitPackage(surfaceNamer.publicClassName(responseName));
+      return typeTable.getAndSaveNicknameFor(fullName.getFullName());
+    } else {
+      return typeTable.getAndSaveNicknameFor("com.google.protobuf.Empty");
```

Comment:
Why is there a protobuf type name in `DiscoveryMethodModel`?

Reply:
Replaced `Empty` with `java.lang.Void`. This represents an empty type for the *Callable generic classes.